setup.py

```diff
@@ -12,7 +12,7 @@
         'pytz~=2020.1',
         'graphviz==0.4.10',
         'jsonschema~=3.2',
-        'gdcdictionary @ git+https://github.com/NCI-GDC/gdcdictionary.git@2.2.0#egg=gdcdictionary',
+        'gdcdictionary @ git+https://github.com/NCI-GDC/gdcdictionary.git@b37ac58bd50e4d04c542b85e03bb9eb7e92ea7c3#egg=gdcdictionary',
```

Comment:
It looks like there isn't a new tag for this yet. I think we should wait until we get a tag before merging this.